wordvecspace/server.py

```diff
@@ -212,10 +219,10 @@ def __init__(self, _type, input_file, port, n_trees=N_TREES, metric=METRIC, inde
     def start(self):
         self.api = API(log=self.log)
         self.api.register(APIFunctions(self._type,
```

Comment:
@prasannababuAddagiri here we can pass all arguments in single line no need to split them.